Content.Server/Prayer/PrayerSystem.cs

```diff
@@ -69,7 +69,7 @@ public void SendSubtleMessage(IPlayerSession target, string messageString, strin
             return;
 
         _popupSystem.PopupEntity(popupMessage, target.AttachedEntity.Value, Filter.Empty().AddPlayer(target), PopupType.Large);
-        _chatManager.ChatMessageToOne(ChatChannel.Local, messageString, popupMessage + " \"{message}\"", EntityUid.Invalid, false, target.ConnectedClient);
+        _chatManager.ChatMessageToOne(ChatChannel.Local, messageString, popupMessage + $" \"{messageString}\"", EntityUid.Invalid, false, target.ConnectedClient);
```

Comment:
Nice meme, popupMessage also means a NAME of the voice in the head

Reply:
isn't it supposed to be $message since chat does some localization stuff

Reply:
I was looking in the game and saw literally "god: {message}"